### ISSUE TITLE
[Suggestion] Ghost View when Wiring Logic Blocks

### ISSUE THREAD

**Issue by XRey360** [tsunamayo/Skywanderers#71]
The logic system is amazing. It's compact, it's functional, we all love it already. But it can be improved even more if all the non-interactive blocks turn semi-transparent when you are wiring something, allowing a better vision of what can be connected and where it is located. Or, alternatively, make the interactive blocks outlined and visible through anything else.

**Comment by tsunamayo** [comment]
Yes that is a still a valid and pertinent request, I will see what I can do. Thanks!
